src/components/LatestBlogs/index.tsx

```
@@ -0,0 +1,57 @@
+export default function LatestBlogs() {
+  return (
+    <div className="max-w-7xl px-4 py-10 sm:px-6  lg:py-14 mx-auto">
+      <div className="max-w-2xl text-center mx-auto mb-10 lg:mb-14">
+        <h2 className="text-3xl font-bold md:text-4xl md:leading-tight dark:text-white">Read our latest blogs</h2>
+        <p className="mt-1 text-gray-600 dark:text-neutral-400">We've works tirelessly for the last 2 years, let us share some insight</p>
+      </div>
+
+      <div className="grid sm:grid-cols-1 md:grid-cols-3 xl:grid-cols-4 gap-6 mb-10 lg:mb-14">
+
+        {blogs.map((blog) => (
+          <BlogCard {...blog} />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)